README.md

```diff
@@ -1 +1,250 @@
+
+</div>
+
+
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+
+
+## 📖 To-Do list <a name="about-project"></a>
+
+
+This is a  website that displays a list of Todo tasks and allows you to add new task, adit task and delete a completed task from the list
+
+
+## 🛠 Built With  VSCode  <a name="Built With VSCode"></a>
+
+### HTML/CSS, javascript, Webpack <a name="tech-stack"></a>
+
+>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">HTML</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Styling</summary>
+  <ul>
+    <li><a href="https://expressjs.com/">CSS</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>functionality</summary>
+ 
+</details>
+
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> A beginner developer can use this template as a starting point for creating one page APP projects using Javascript and HTML/CSS with clean and organized code using the module directory.
+
+
+
+### Prerequisites
+
+In order to run this project you need:
+- Node
+- ESlint set up
+
+
+<!--
+Example command:
+
+```sh
+ gem install rails
+```
+ -->
+
+### Setup
+
+Clone this repository to your desired folder:
+<br>
+<br>gh repo clonehttps://github.com/Tafloninno/To_do_list
+
+
+
+### Install
+
+Install this project with:
+
+
+
+```sh
+  
+  gh repo clone Tafloninno/To_do_list
+  cd To_do_list
+  npm install -y
+```
+
+
+
+
+### Run tests
+
+To run tests, run the following command:
+
+
+
+```sh
+      npx stylelint "**/*.{css,scss}"
+
+```
+
+```sh
+      npx hint .
+```
+
+```sh
+    npx eslint .
+```
+
+
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Mugisha.I , <a name="authors"></a>
+
+
+
+👤 **Mugisha.I**
+
+- GitHub: [@githubhandle](https://github.com/Tafloninno)
+
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/mugisha-innocent-19b700238/)
+
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+> I'll add the following section to the webpage.
+
+- [ ] **Add Remove and Add buttons**
+- [ ] **Interactive list**
+
```

Comment:
Update README.md file for the futures that you already have and remove them from your future features!